geometry-constructor/Qt models/main.qml

```diff
@@ -1,130 +1,163 @@
 import QtQuick 2.11
 import QtQuick.Controls 2.4
+import QtQuick.Scene3D 2.0
 import Qt.labs.platform 1.0
 import MyWriters 1.0
 import MyModels 1.0
 
 Rectangle {
-    width: 400; height: 300
+    width: 700
+    height: 300
 
-    Row {
-        id: textRow
+    Rectangle {
```

Comment:
`main.qml` is getting large, could you move everything which is in the main panel into its own qml file?